client/components/prestations/prestation.component.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+angular.module('impactApp')
+  .component('prestation', {
+    templateUrl: 'components/prestations/prestation.html',
+    bindings: {
+      prestation: '=ngModel',
+      titre: '@',
```

Comment:
The `<` symbol denotes one-way bindings which are available since 1.5.
https://docs.angularjs.org/guide/component